src/domains/chart/components/chart-with-loader/chart-with-loader.tsx

```diff
@@ -251,6 +252,8 @@ export const ChartWithLoader = ({
           after: after || null,
           before: before || null,
           dimensions: attributes.dimensions,
+          aggrMethod: attributes.aggrMethod,
```

Comment:
for choosing the aggregation method, there is an old `group` param. Do you know why `aggrMethod` is used instead of `group`?

https://learn.netdata.cloud/docs/agent/web/gui/custom#data-grouping-method

Reply:
This is important. So what is exactly `group` used for? 
Perhaps we need to contact @netdata/cloud-be and use that existing one

Reply:
from what i understand `group` is used for the same thing, different aggregation methods :) Users even not can change that param in their custom `dashboard_info.js`

Reply:
From the above documentation is seems that the grouping method  (`group` param) is used when the Netdata server is requested to return fewer points for a time-frame. 
aggrMethod is for aggregation across room multi nodes...

Reply:
Sorry, i should have wrote `method` param. It's used for the same thing. `group` is the attribute name.

Reply:
It is used for the very same thing, perhaps we should have the same naming on BE side (ie. use `method` URL param)

Reply:
We talked on slack and it turns out I misunderstood, `aggrMethod` is for choosing grouping method across nodes, whereas `method` is for grouping across time, in single node. Naming is a little confusing, but perhaps comments will be enough...